app/controllers/items_controller.rb

```diff
@@ -6,6 +6,8 @@ def index
       @items = @merchant.items
     else
       @items = Item.active_items
+      @popular_items = Item.most_popular_items
```

Comment:
I like that the controller is just holding the array of popular items from the model method!